### PR TITLE
CSV Report. Connected to edenlabllc/ehealth.api#1504 

### PR DESCRIPTION
**Pull request by LehaMotovilov**
Related to: https://github.com/edenlabllc/ehealth.api/issues/1504

### CODE REVIEW

File: app/server/internal-api/reimbursementReportDownload.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  /**
+   * @link https://www.npmjs.com/package/request
+   */
+  request(options, (error, response, body) => {
+    if (
+      response.statusCode === 200 &&
+      response.headers['content-type'] === 'text/csv; charset=utf-8'
+    ) {
+      res.setHeader('content-type', 'text/csv; charset=utf-8');
+      res.send(body);
```
Comment (kiurchv) [inline app/server/internal-api/reimbursementReportDownload.js]: I'm concerned about storing whole report body in the variable. I'll investigate the ability to stream data from API through nhs.admin.web backend and provide guide or proof of concept if will be clarified that this approach is possible

Comment (LehaMotovilov) [reply app/server/internal-api/reimbursementReportDownload.js]: Ok, but we have deadline tomorrow. 🤓

Comment (kiurchv) [reply app/server/internal-api/reimbursementReportDownload.js]: So, let's merge this if I'll not found the solution until tomorrow, but we should take into consideration this issue anyway

Comment (LehaMotovilov) [reply app/server/internal-api/reimbursementReportDownload.js]: Ok, thanks.

**Comment by LehaMotovilov** [comment]
@ynotLeft @ivchenkodima guys, any comments?

**Pull request merged**